File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/duckdb/NullableTypes.scala
package dev.typr.foundationssc.docs.duckdb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object NullableTypes:
  //start
  val notNull: DuckDbType[Int] = DuckDbTypes.integer
  val nullable: DuckDbType[Option[Int]] = DuckDbTypes.integer.opt
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/duckdb/NullableTypes.kt
package dev.typr.foundationskt.docs.duckdb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class NullableTypes {
    //start
    val notNull: DuckDbType<Int> = DuckDbTypes.integer
    val nullable: DuckDbType<Int?> = DuckDbTypes.integer.opt()
    //stop
}
